libs/shared/db/mysql/account/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+export { AccountCustomers } from './lib/account-customers';
```

Comment:
This is fine. I'll often see something like `export * from './lib/account-customers' as well. Seems less specific but maybe less prone to forgetting an export down the road.